### ISSUE TITLE
Pre-work Checklist: Developer: [Edward Salazar]

### ISSUE THREAD

**Issue by EdASalazar** [hackforla/website#4336]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [ ] (Optional) Office Hours, Thursdays 7-8pm PST
  - [ ] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues in the [prioritized backlog](https://github.com/hackforla/website/projects/7#column-7198257) only with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by EdASalazar** [comment]
I've done my First Issues and I'm waiting for a Medium Issue to show up so I can continue doing my Pre-Work.

**Comment by EdASalazar** [comment]
Progress: I'm about to update my Medium pull request. That should get merged. (fingers crossed)
Blockers: After that, I'll start looking for  Complexity: Large. 
Availability: 6-12 hours next week 
ETA: Hopefully this week I can find Complexity: Large issue.

**Comment by EdASalazar** [comment]
My medium issue was merged. Moving on to Large.

**Comment by EdASalazar** [comment]
I'm finishing a Medium issue and when I'm done with that I'll try to find a Large issue.

**Comment by EdASalazar** [comment]
I have a Large complexity issue. I'm starting to work on that. Hopefully, I can resolve it this week. That should complete my Pre-Work Checklist.

**Comment by roslynwythe** [comment]
Hi @EdASalazar.  Congratulations on completing your `Complexity: Medium` issue.  During periods when you are not working on any issues, we request that you post a comment in this issue to inform us of your progress, any problems you might be having, and to make sure that you are still interested in continuing with Hack for LA.  We would like to see updates, in this issue or in other issues, at least every two weeks.  

I noticed that you unassigned from an issue recently.  In the future, please notify a dev lead if you wish to stop work on an issue.  It's not a problem at all, its just that we want to know about it so that be aware that the issue is not currently in progress and so can prepare the issue for someone else.  Thanks!

**Comment by EdASalazar** [comment]
Sorry about that. I'll make sure to notify someone going forward. 

@roslynwythe I spoke with Will but I can add some notes here. 

I'm doing the FDA TOP hackathon. I think it's going to eat up any free time I have.  It should be done by mid-August. Until then I'll need to pause my work here. 

I plan on keeping up with the meetings so I don't lose too much momentum. If that's allowed.

**Comment by EdASalazar** [comment]
I'm currently part of an FDA Hackathon program. That should wrap up around 8/9 and then I'll continue to finish up my pre-work checklist.

**Comment by ExperimentsInHonesty** [comment]
@EdASalazar I see you never did a large size issue.  If you are still interested in working on issues on the Hack for LA website team, here is the link to [large issues, in the prioritized backlog](https://github.com/hackforla/website/projects/7?card_filter_query=-label%3A%22role%3A+design%22+-label%3A%22role%3A+dev+leads%22+-label%3A%22role%3A+user+research%22+label%3A%22complexity%3A+large%22#column-7198257)

In the meantime, I am closing this prework issue.

**Comment by HackforLABot** [comment]
## Activity Log: EdASalazar
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- EdASalazar assigned: https://github.com/hackforla/website/issues/3870 at 03/28/2023, 09:11 PM PDT
- EdASalazar opened: https://github.com/hackforla/website/issues/4336 at 03/28/2023, 09:22 PM PDT
- EdASalazar unassigned: https://github.com/hackforla/website/issues/3870#issuecomment-1487919594 at 03/28/2023, 09:24 PM PDT
- EdASalazar assigned: https://github.com/hackforla/website/issues/4336 at 03/28/2023, 09:28 PM PDT
- EdASalazar assigned: https://github.com/hackforla/website/issues/4012#issuecomment-1484410244 at 04/10/2023, 06:19 PM PDT
- EdASalazar commented on: https://github.com/hackforla/website/issues/4012#issuecomment-1502565076 at 04/10/2023, 06:41 PM PDT
- EdASalazar opened: https://github.com/hackforla/website/pull/4465 at 04/10/2023, 07:07 PM PDT
- EdASalazar commented on: https://github.com/hackforla/website/pull/4465#issuecomment-1503767366 at 04/11/2023, 09:51 AM PDT
- EdASalazar PR was merged: https://github.com/hackforla/website/pull/4465#event-8977183395 at 04/11/2023, 11:54 AM PDT
- EdASalazar opened: https://github.com/hackforla/website/issues/4486 at 04/11/2023, 08:10 PM PDT
- EdASalazar assigned: https://github.com/hackforla/website/issues/4393 at 04/12/2023, 10:24 PM PDT
- EdASalazar commented on: https://github.com/hackforla/website/issues/4393#issuecomment-1506365948 at 04/12/2023, 10:25 PM PDT
- EdASalazar opened: https://github.com/hackforla/website/pull/4500 at 04/13/2023, 07:05 AM PDT
- EdASalazar PR was merged: https://github.com/hackforla/website/pull/4500#event-9002657356 at 04/13/2023, 06:33 PM PDT
- EdASalazar issue closed by PR 4500: https://github.com/hackforla/website/issues/4393#event-9002657458 at 04/13/2023, 06:33 PM PDT
- EdASalazar commented on: https://github.com/hackforla/website/issues/4336#issuecomment-1521009883 at 04/24/2023, 05:48 PM PDT
- EdASalazar assigned: https://github.com/hackforla/website/issues/4574 at 05/02/2023, 05:39 PM PDT
- EdASalazar commented on: https://github.com/hackforla/website/issues/4574#issuecomment-1532318414 at 05/02/2023, 05:43 PM PDT
- EdASalazar commented on: https://github.com/hackforla/website/issues/4574#issuecomment-1533913727 at 05/03/2023, 05:31 PM PDT
- EdASalazar commented on: https://github.com/hackforla/website/issues/4574#issuecomment-1535787623 at 05/04/2023, 11:42 PM PDT
- EdASalazar opened: https://github.com/hackforla/website/pull/4603 at 05/05/2023, 12:25 AM PDT
- EdASalazar commented on: https://github.com/hackforla/website/issues/4012#issuecomment-1539196424 at 05/08/2023, 04:41 PM PDT
- EdASalazar unassigned: https://github.com/hackforla/website/issues/4012#issuecomment-1539196424 at 05/08/2023, 05:21 PM PDT
- EdASalazar commented on: https://github.com/hackforla/website/pull/4603#issuecomment-1540892121 at 05/09/2023, 02:07 PM PDT
- EdASalazar commented on: https://github.com/hackforla/website/pull/4603#issuecomment-1541247590 at 05/09/2023, 07:57 PM PDT
- EdASalazar commented on: https://github.com/hackforla/website/pull/4603#issuecomment-1544802991 at 05/11/2023, 04:01 PM PDT
- EdASalazar commented on: https://github.com/hackforla/website/issues/4336#issuecomment-1546586634 at 05/13/2023, 12:35 AM PDT
- EdASalazar commented on: https://github.com/hackforla/website/pull/4603#issuecomment-1546588238 at 05/13/2023, 12:45 AM PDT
- EdASalazar PR was merged: https://github.com/hackforla/website/pull/4603#event-9239444298 at 05/13/2023, 07:04 PM PDT
- EdASalazar issue closed by PR 4603: https://github.com/hackforla/website/issues/4574#event-9239444309 at 05/13/2023, 07:04 PM PDT
- EdASalazar commented on: https://github.com/hackforla/website/issues/4336#issuecomment-1546986512 at 05/14/2023, 12:45 PM PDT
- EdASalazar issue closed as completed: https://github.com/hackforla/website/issues/4012#event-9241187725 at 05/14/2023, 06:28 PM PDT
- EdASalazar assigned: https://github.com/hackforla/website/issues/4012#event-9241187725 at 05/14/2023, 06:28 PM PDT
- EdASalazar assigned: https://github.com/hackforla/website/issues/4659 at 05/23/2023, 07:34 PM PDT
- EdASalazar commented on: https://github.com/hackforla/website/issues/4659#issuecomment-1560376077 at 05/23/2023, 07:36 PM PDT
- EdASalazar commented on: https://github.com/hackforla/website/issues/4659#issuecomment-1565249991 at 05/27/2023, 12:23 AM PDT
- EdASalazar commented on: https://github.com/hackforla/website/issues/4336#issuecomment-1567716513 at 05/29/2023, 08:42 PM PDT
- EdASalazar opened: https://github.com/hackforla/website/pull/4755 at 05/29/2023, 09:33 PM PDT
- EdASalazar submitted review: https://github.com/hackforla/website/pull/4750#pullrequestreview-1453923568 at 05/31/2023, 11:44 AM PDT
- EdASalazar commented on: https://github.com/hackforla/website/pull/4755#issuecomment-1571105830 at 05/31/2023, 04:52 PM PDT
- EdASalazar commented on: https://github.com/hackforla/website/pull/4755#issuecomment-1576006459 at 06/04/2023, 09:04 PM PDT
- EdASalazar commented on: https://github.com/hackforla/website/pull/4755#issuecomment-1577820033 at 06/05/2023, 07:55 PM PDT
- EdASalazar commented on: https://github.com/hackforla/website/pull/4755#issuecomment-1579275181 at 06/06/2023, 11:42 AM PDT
- EdASalazar commented on: https://github.com/hackforla/website/pull/4755#issuecomment-1579574171 at 06/06/2023, 04:03 PM PDT
- EdASalazar commented on: https://github.com/hackforla/website/pull/4755#issuecomment-1579580221 at 06/06/2023, 04:11 PM PDT
- EdASalazar assigned: https://github.com/hackforla/website/issues/4189 at 06/07/2023, 11:20 AM PDT
- EdASalazar commented on: https://github.com/hackforla/website/issues/4189#issuecomment-1581305348 at 06/07/2023, 11:22 AM PDT
- EdASalazar PR was merged: https://github.com/hackforla/website/pull/4755#event-9480020370 at 06/08/2023, 09:04 PM PDT
- EdASalazar issue closed by PR 4755: https://github.com/hackforla/website/issues/4659#event-9480020618 at 06/08/2023, 09:04 PM PDT
- EdASalazar commented on: https://github.com/hackforla/website/issues/4336#issuecomment-1586401764 at 06/11/2023, 05:39 PM PDT
- EdASalazar commented on: https://github.com/hackforla/website/issues/4189#issuecomment-1587812718 at 06/12/2023, 10:58 AM PDT
- EdASalazar commented on: https://github.com/hackforla/website/issues/4189#issuecomment-1598260931 at 06/20/2023, 12:28 AM PDT
- EdASalazar commented on: https://github.com/hackforla/website/issues/4189#issuecomment-1601255747 at 06/21/2023, 10:10 AM PDT
- EdASalazar unassigned: https://github.com/hackforla/website/issues/4189 at 06/26/2023, 11:01 AM PDT
- EdASalazar commented on: https://github.com/hackforla/website/issues/4336#issuecomment-1624417300 at 07/06/2023, 04:09 PM PDT
- EdASalazar commented on: https://github.com/hackforla/website/issues/4336#issuecomment-1656426197 at 07/28/2023, 04:07 PM PDT
